src/Dapr.Actors/Runtime/ConverterUtils.cs

```diff
@@ -68,5 +71,69 @@ public static string ConvertTimeSpanValueInDaprFormat(TimeSpan? value)
 
             return stringValue;
         }
+        
+        public static string ConvertTimeSpanValueInISO8601Format(TimeSpan value, int repetitions)
+        {
+            if (repetitions <= 0)
+            {
+                return ConvertTimeSpanValueInDaprFormat(value);
+            }
+
+            StringBuilder builder = new StringBuilder();
+            builder.AppendFormat("R{0}/P", repetitions);
```

Comment:
Are years/months specifically not supported?

Reply:
Nope. From the docs:

    // TimeSpan is internally represented as a number of milliseconds.  While
    // this maps well into units of time such as hours and days, any
    // periods longer than that aren't representable in a nice fashion.
    // For instance, a month can be between 28 and 31 days, while a year
    // can contain 365 or 364 days.  A decade can have between 1 and 3 leapyears,
    // depending on when you map the TimeSpan into the calendar.  This is why
    // we do not provide Years() or Months().